questions_page/questions_page.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Questions page</title>
+    
+    <link
+      rel="stylesheet"
+      href="https://clean-u-i.netlify.app/component/component.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+    />
+    <link rel="stylesheet" href="/rules_page/rules_page.css" />
+    <link rel="stylesheet" href="/index.css" />
+    <link rel="stylesheet" href="/common.css" />
+    <link rel="stylesheet" href="/questions_page/questions_page.css" />
+  </head>
+  <body>
+    <div class="question-page-container">
+        <div class="header flex-row justify-between align-center px-2">
+            <div class="flex-row justify-center align-center">
+                <img class ="logo-image" src="/images/logo3.png" alt="logo"/>
+                <span class="logo-text font-semibold">Quiz<span class="text-orange">Buzz</span></span>
+            </div>
+            <div class="search-bar-container">
+              <button type="submit">
```

Comment:
you may remove type="submit" if not in use

Reply:
made changes in it!!